indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -139,6 +142,7 @@
                         url: url,
                         method: method,
                         data: params,
+                        content: $this.parent().find(content).html(),
```

Comment:
What is the likelihood the content will be far away, in a completely different part of the DOM tree? But yeah, maybe this is too much...